merlin/dag/selector.py

```diff
@@ -46,6 +46,12 @@ def __init__(
         self._tags = tags if tags is not None else []
         self.subgroups = subgroups if subgroups is not None else []
 
+        self.all = names == "*"
```

Comment:
In order to make it possible to select all columns by default, which is useful in order to make the `ComputeSchemaMixin` methods have the lightest possible number of required parameters, we introduce the wildcard selector `"*"`, which is used in one of the following ways:
```
ColumnSelector("*")
"*" >> SomeOperator()
```

Reply:
This is also useful for defining operator graphs that can be run on different input data by the same executor, since it allows late-binding to all provided columns in a similar way to how tags allow late-binding to some provided columns (but not others.)